test/api/api_test.go

```diff
@@ -164,34 +166,47 @@ func waitForAPI(apiUrl string) error {
 func RunApiServer(t *testing.T, binary string, storageApiHost string, repoPath string) string {
 	t.Helper()
 
+	// Get a free port
 	port, err := getFreePort()
 	if err != nil {
 		t.Fatalf("Could not receive a free port: %s", err)
 	}
 
+	// Args
 	apiUrl := fmt.Sprintf("http://localhost:%d", port)
 	args := []string{fmt.Sprintf("--http-port=%d", port)}
 	if repoPath != "" {
 		args = append(args, fmt.Sprintf("--repositories=keboola|file://%s", repoPath))
 	}
+
+	// Envs
+	envs := env.Empty()
+	envs.Set("PATH", os.Getenv("PATH"))
+	envs.Set("KBC_STORAGE_API_HOST", storageApiHost)
+	envs.Set("DATADOG_ENABLED", "false")
+	envs.Set("ETCD_ENABLED", "false")
```

Comment:
Envs pre spustenie API binarky, `PATH` je potrebne pre to, aby sa nasiel `git` command.